### DISCUSSION TITLE
How to force q-tab-panel to fill all height of its parent.

### DISCUSSION THREAD

**Discussion by its-saeed** [quasarframework/quasar#12660] [CLI - PWA mode] [answered]
I want to force that yellow q-tab-panel to fill the whole height of its parent.

![tab](https://user-images.githubusercontent.com/5278422/156835491-1540e6fc-b55c-414b-a4dc-e070db2352b2.png)

**Comment by MilosPaunovic** [comment]
```html
<QTabPanel class="full-height" />
```

**Reply by its-saeed** [reply]
nothing changed

**Reply by MilosPaunovic** [reply]
Create a Codepen reproduction of your issue so it could be played with.

**Comment by its-saeed** [comment, marked as answer]
[This post](https://forum.quasar-framework.org/topic/5315/q-scroll-area-height) helped me to fix the problem. I should have set a fixed height for parent.
